src/app/checkout/step-2/credit-card/credit-card.tpl.html

```diff
@@ -1,124 +1,153 @@
-                                    <select class="form-control  form-control-subtle" id="suffix">
-                                        <option>Peoples Republic of China</option>
-                                    </select>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                    <div class="row">
-                        <div class="col-sm-12">
-                            <div class="form-group">
-                                <label>Address</label>
-                                <input type="text" name="address1" class="form-control  form-control-subtle">
-                            </div>
-                        </div>
-                    </div>
-                    <div class="row">
-                        <div class="col-sm-12">
-                            <div class="form-group">
-                                <input type="text" name="address2" class="form-control  form-control-subtle">
-                            </div>
-                        </div>
-                    </div>
-                    <div class="row">
-                        <div class="col-sm-12">
-                            <div class="form-group">
-                                <label>City</label>
-                                <input type="text" class="form-control  form-control-subtle">
-                            </div>
-                        </div>
-                    </div>
-                    <div class="row">
-                        <div class="col-sm-6">
-                            <div class="form-group">
-                                <label>State</label>
-                                <div class="form-group">
-                                    <select class="form-control  form-control-subtle" id="suffix">
-                                        <option></option>
-                                    </select>
-                                </div>
-                            </div>
-                        </div>
-                        <div class="col-sm-6">
-                            <div class="form-group">
-                                <label>Zip</label>
-                                <input type="text" class="form-control  form-control-subtle">
-                            </div>
-                        </div>
-                    </div>
+      <div class="panel panel-default">
+        <div class="panel-body">
+          <div class="mb">
+            <h4 class="panel-title  border-bottom-small  visible">New Billing Address</h4>
+            <div class="row">
+              <div class="col-sm-12">
+                <div class="form-group">
+                  <label>Country</label>
+                  <div class="form-group">
+                    <select class="form-control  form-control-subtle" id="suffix">
+                      <option>Peoples Republic of China</option>
```

Comment:
I actually haven't done anything with the billing address yet. It is all hidden currently. I guess I should work on that next. I think I'd prefer to tackle that in another PR.